src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -152,20 +152,22 @@ const ExpandedNodeField: FC<ExpandedNodeFieldProps> = ({ title, description }) =
 type StatusFieldProps = {
   documentation: DocumentationResource | undefined;
   isAccepted: boolean | null;
+  isGuidance: boolean;
 };
 
-const StatusField: FC<StatusFieldProps> = ({ documentation, isAccepted }) => {
+const StatusField: FC<StatusFieldProps> = ({ documentation, isAccepted, isGuidance }) => {
   if (!documentation?.resource) {
     return null;
   }
   const status = documentation.status;
   const rawDate = documentation.resource?.meta?.lastUpdated;
   if (rawDate) {
     const date = new Date(rawDate).toLocaleString('en-us');
+    const declinedText = isGuidance ? 'Declined' : 'Status';
     return (
       <ExpandedNodeField
         key="Status"
-        title={isAccepted ? status.charAt(0).toUpperCase() + status.slice(1) : 'Declined'}
+        title={isAccepted ? status.charAt(0).toUpperCase() + status.slice(1) : declinedText}
```

Comment:
Is there even possibly a simpler approach? `isAccepted` is `boolean | null`, so I think it's that `null` case that is tripping up the logic here. Extra parameters might not even be needed

| `isAccepted` | expected result |
|--------------|---------|
| `true` | "Accepted" <br/>(or I guess the `status` field from the resource like we already have) |
| `false` | "Declined" |
| `null` | "Status" |